packages/icons/package.json

```
@@ -0,0 +1,74 @@
+{
+  "name": "@vibe/icons",
+  "version": "0.0.4",
+  "description": "Vibe's icon library",
+  "repository": "git+https://github.com/mondaycom/vibe.git",
```

can you please change to this?
```suggestion
  "repository": {
    "type": "git",
    "url": "git+https://github.com/mondaycom/vibe.git",
    "directory": "packages/icons"
  },
```